openshift/hourlies-prune/docker/prune_hourlies.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# usage example:
+# AWS_HOSTNAME=[your aws hostname] AWS_ACCESS_KEY=[your access key] AWS_SECRET_KEY=[your secret key] AWS_BUCKET=[your aws bucket] ./prune_hourlies.sh
+
+# variable checks
+
+if [ -z ${AWS_HOSTNAME+0} ]
+then
+    echo "AWS_HOSTNAME not specified"
+    echo "Specify an AWS hostname"
+    exit 1
+fi
+
+if [ -z ${AWS_ACCESS_KEY+0} ]
+then
+    echo "AWS_ACCESS_KEY not specified"
+    echo "Specify an AWS access key"
+    exit 1
+fi
+
+if [ -z ${AWS_SECRET_KEY+0} ]
+then
+    echo "AWS_SECRET_KEY not specified"
+    echo "Specify an AWS secret key"
+    exit 1
+fi
+
+if [ -z ${AWS_BUCKET+0} ]
+then
+    echo "AWS_BUCKET not specified"
+    echo "Specify an AWS bucket"
+    exit 1
+fi
+
+hourlies_path="sfms/uploads/hourlies"
+day_before_yesterday=$(TZ=America/Vancouver date -v-2d -I)
```

Comment:
Grabs the date in ISO 8601 and subtracts 2 days from it to get the day before yesterday